### ISSUE TITLE
Migrations on other servers (not development) Question

### ISSUE THREAD

**Issue by Allann** [dotnet/efcore#7182]
### The issue
When development is finished and TFS (VSTS) takes over.  The build completes successfully and the artifact directory contains the published website via a build definition.  The Release Manager is then triggered, and takes over and creates a website in an environment, the website files are copied to the environment and the website started. All good so far.

Missing the Database migration steps here...

Now someone starts the server and it fails because the database has not been migrated. How do we do that? The EF Tools were not copied to that environment, in fact we can't even run dotnet.exe because the core sdk has not been deployed onto that environment either. so even if we had the tools, we can't execute them.

In EF 6.x it was easy cause we could just copy migrate.exe as part of the artifacts, and run that on the environment after the files were copied. As I understand it, the EF Tools are now a DLL (ef.dll) that has to be run by dotnet.exe. Does this mean that all web servers now need Core SDK to perform migrations?

Other details about my project setup:
asp.net.core application being deployed onto a QA server for testing.  This is not a dev box, does not have VS2015 installed, does not have Core SDK installed.

**Comment by ErikEJ** [comment]
The team plans to create EF.exe

**Comment by Allann** [comment]
I found an ef.exe in Preview 4.  Is that the one to replace migrate.exe?  If so, is there an expected way to get that from the package to the artefacts directory?  For a test today, I manually dragged it into a "tools" directory in the solution and used a copy files task in the build server to add it to the artefacts.  Still need a way to run it on the remote server from the release task.  I'm thinking a powershell on remote pc task should be able to achieve that?
